src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -75,6 +75,16 @@ public LoanRepository(Clients clients) {
       .thenApply(mapResult(loanAndRelatedRecords::withLoan));
   }
 
+  public CompletableFuture<Result<Loan>> updateLoanAndItemInStorage(Loan loan) {
```

Comment:
I don’t think the loan repository should be coordinating the update of both the loan and item together. Can this be done higher up in the process, like I think the other processes do?

Reply:
Done.